src/hooks/useBasisForm/validation/getBasisErrors.ts

```diff
@@ -0,0 +1,14 @@
+import { MultipleFieldErrors } from "react-hook-form";
+import { ComponentName } from "../../../components/componentName";
+import { ValidationError } from "../../../types";
+
+const isNonEmptyString = (item: any): item is string => typeof item === "string" && !!item;
```

Comment:
Is this the right place to add the default input validation where if there blanks (spaces) only the input considered invalid?